charts/legend/src/Legend/Legend.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { forwardRef } from 'react';
+import { useSeriesContext } from '@lg-charts/series-provider';
+
+import { LegendCheckbox } from '../LegendCheckbox';
+
+import { getLegendStyles } from './Legend.styles';
+import { LegendProps } from './Legend.types';
+
+export const Legend = forwardRef<HTMLDivElement, LegendProps>(
+  (
+    { className, series, seriesNameFormatter, showSelectAll = true, ...rest },
+    fwdRef,
+  ) => {
+    const {
+      isChecked,
+      isSelectAllChecked,
+      isSelectAllIndeterminate,
+      toggleSeries,
+      toggleSelectAll,
+    } = useSeriesContext();
+
+    return (
+      <div {...rest} className={getLegendStyles({ className })} ref={fwdRef}>
```

Comment:
Do you think we need a way to indicate, for accessibility reasons, that all these checkboxes control the lines in the charts? Maybe a hidden label with descriptive text?